src/middlewared/middlewared/alembic/versions/22.02/2022-08-24_12-58_update_iscsi_zvol_path.py

```diff
@@ -0,0 +1,33 @@
+"""Update iscsi zvol path
+
+Revision ID: 98b2cfaa0e5a
+Revises: 7c8da45b515e
+Create Date: 2022-08-24 12:58:02.805097+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '98b2cfaa0e5a'
+down_revision = '7c8da45b515e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    # ### commands auto generated by Alembic - please adjust! ###
+    for extent in conn.execute("select id, iscsi_target_extent_path from services_iscsitargetextent"):
+        id, path = extent
+        if path.startswith('zvol/') and ' ' in path:
+            path = path.replace(' ', '+')
+            conn.execute(f'update services_iscsitargetextent set iscsi_target_extent_path="{path}" where id={id}')
```

Comment:
There is an obscure but still possible SQL injection vulnerability here :D
Low score but things we need to start thinking about as we add RBAC / API ACL.